routers/repo/setting_protected_branch.go

```diff
@@ -125,6 +126,15 @@ func SettingsProtectedBranch(c *context.Context) {
 	c.Data["whitelist_users"] = strings.Join(base.Int64sToStrings(protectBranch.WhitelistUserIDs), ",")
 	c.Data["merge_whitelist_users"] = strings.Join(base.Int64sToStrings(protectBranch.MergeWhitelistUserIDs), ",")
 	c.Data["approvals_whitelist_users"] = strings.Join(base.Int64sToStrings(protectBranch.ApprovalsWhitelistUserIDs), ",")
+	c.Data["branch_status_check_contexts"], _ = models.FindRepoStatusCheckContexts(c.Repo.Repository.ID, 4*7*24*time.Hour) // Find last week status check contexts
```

Comment:
Isn't `4*7*24*time.Hour` a month and not a week?

Reply:
changed to 7*24*time.Hour